cmd/rollouts-controller/main.go

```diff
@@ -213,6 +214,7 @@ func newCommand() *cobra.Command {
 	command.Flags().Int64Var(&rolloutResyncPeriod, "rollout-resync", controller.DefaultRolloutResyncPeriod, "Time period in seconds for rollouts resync.")
 	command.Flags().BoolVar(&namespaced, "namespaced", false, "runs controller in namespaced mode (does not require cluster RBAC)")
 	command.Flags().StringVar(&logLevel, "loglevel", "info", "Set the logging level. One of: debug|info|warn|error")
+	command.Flags().StringVar(&logFormat, "logformat", "info", "Set the logging format. One of: text|json")
```

Comment:
This is set to `info` which is not a valid option. I think default should be empty string so that behavior is whatever logrus' default behavior is.

Reply:
Good catch! I have set it to an empty string.